quick-start/src/main/resources/static/app/mlcp/mlcp.controller.js

```diff
@@ -157,7 +177,9 @@
     }
 
     $scope.$watch('mlcp.input_file_path', function(value) {
-      searchPath(value);
+      if(isValidInputFilePath(value)) {
```

Comment:
The first time you open this dialog ever you get the error right off the bat. We need to set a default value for input_file_path or something.